### PR TITLE
add Svelte landing page

### PR DESCRIPTION
**Pull request by marcoow**
## TODO

- [x] update metadata (title, description, etc.)
- [x] update header section content
- [x] update first content block content
- [x] remove EuroRust section
- [x] remove company logos
- [x] fix display of blog posts (for fewer than 4 posts, we should not show the large one and then 3 below)

### CODE REVIEW

File: src/svelte-consulting.njk [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  </div>
+</section>
+
+<section class="split-content">
+  <div class="container container--lg">
+    <h2 class="h5">Svelte & Mainmatter</h2>
+    <h3 class="split-content__subheading h2">A unique skillset</h3>
+    <div class="split-content__content">
+      <div class="split-content__text">
+        <p>We’re frontend experts - our team speaks at international events, is <a href="/this-week-in-open-source/">actively contributing to a number of open source projects</a> and our Svelte expert Paolo Ricciuti is the co-author of <a href="https://www.sveltelab.dev/">SvelteLab</a>.</p>
```
Comment (paoloricciuti) [inline src/svelte-consulting.njk]: I think adding that I'm a svelte ambassador can be beneficial

**Comment by KevinBongart** [comment]
FYI, Paolo's bio is full-width unlike the other bios:

Screenshot 2023-09-26 at 15 52 33

**Comment by SaeLLo** [comment]
Bios:
> not consistent: Not all have the twitter and LinkedIn button
> When clicking on Gabors name it says "Page not Found"

File: src/svelte-consulting.njk [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        {% include 'content/secondary-feature-svelte.njk' %}
+      </div>
+    </div>
+  </div>
+</section>
+
+{% set 'slide_1' = {
+  "color": "purple",
+  "title": "Succeeding with Svelte",
+  "text": "Getting started with Svelte and SvelteKit is straight-forward but an experienced partner can help ensure you're set up for success from the beginning. Paolo Ricciuti, co-author of <a href=\"https://www.sveltelab.dev\">SvelteLab</a> and our in-house Svelte expert, is here to help your team.",
```
Comment (SaeLLo) [inline src/svelte-consulting.njk]: "Straight-forward" is one word

File: src/svelte-consulting.njk [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  "text": "Svelte has excellent documentation for diving straight into the framework. However, sometimes active learning in a workshop taught by a topic expert can accelerate the learning experience. Our trainers are world-class mentors and can design a learning path that best fits your needs.",
+  "linkUrl": "/services/workshops/svelte-and-sveltekit/",
+  "linkClass": "plausible-event-name=Svelte+Workshop",
+  "linkText": "Check out our Svelte & SvelteKit workshop!"
+} %}
+
+{% set 'slide_3' = {
+  "color": "aqua",
+  "title": "Team augmentation",
+  "text": "Svelte is still a young technology – hiring Senior talent can be a challenge.
```
Comment (SaeLLo) [inline src/svelte-consulting.njk]: Senior talent -> small s or capital T

File: src/svelte-consulting.njk [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  "title": "Succeeding with Svelte",
+  "text": "Getting started with Svelte and SvelteKit is straight-forward but an experienced partner can help ensure you're set up for success from the beginning. Paolo Ricciuti, co-author of <a href=\"https://www.sveltelab.dev\">SvelteLab</a> and our in-house Svelte expert, is here to help your team.",
+  "linkUrl": "https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ26xh3Rs58rD4ZNFuzlzbTxFxubT5_9qSqc_NU4Pmqdot1YTIiylxcyL2kJrfEHn8tpgJqoHi8O",
+  "linkText": "Book a free 1:1 call with Paolo"
+} %}
+
+{% set 'slide_2' = {
+  "color": "black",
+  "title": "Training",
+  "text": "Svelte has excellent documentation for diving straight into the framework. However, sometimes active learning in a workshop taught by a topic expert can accelerate the learning experience. Our trainers are world-class mentors and can design a learning path that best fits your needs.",
```
Comment (SaeLLo) [inline src/svelte-consulting.njk]: "Subject Matter Expert" would be the right term to use here (instead of "topic expert")

**Comment by marcoow** [comment]
> FYI, Paolo's bio is full-width unlike the other bios:

this is built that way actually – not sure why

**Comment by marcoow** [comment]
> not consistent: Not all have the twitter and LinkedIn button

not everyone has all of these services I guess

> When clicking on Gabors name it says "Page not Found"

fixed

File: src/svelte-consulting.njk [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  </div>
+</section>
+
+<section class="split-content">
+  <div class="container container--lg">
+    <h2 class="h5">Svelte & Mainmatter</h2>
+    <h3 class="split-content__subheading h2">A unique skillset</h3>
+    <div class="split-content__content">
+      <div class="split-content__text">
+        <p>We’re frontend experts - our team speaks at international events, is <a href="/this-week-in-open-source/">actively contributing to a number of open source projects</a> and our Svelte expert Paolo Ricciuti is a Svelte ambassador and the co-author of <a href="https://www.sveltelab.dev/">SvelteLab</a>.</p>
```
Comment (SaeLLo) [inline src/svelte-consulting.njk]: We use a bigger "– " in the other segments

**Comment by SaeLLo** [comment]
> > not consistent: Not all have the twitter and LinkedIn button
> 
> not everyone has all of these services I guess

https://www.linkedin.com/in/paolo-ricciuti-571a91187/
https://www.linkedin.com/in/daniel-beer-569b51251/
https://www.linkedin.com/in/pichfl/
https://twitter.com/pichfl

**Review by SaeLLo — APPROVED** [review]

**Comment by marcoow** [comment]
added – I don't think we want to add @beerinho's Twitter tbh :)

**Pull request merged**